server.go

```diff
@@ -791,6 +792,14 @@ func (server *Server) wireFrontendBackend(serverRoute *serverRoute, handler http
 		}
 	}
 
+	// path replace
+	if len(serverRoute.replacePath) > 0 {
```

Comment:
there is no existing test file for server.go, should I make one to only test this part?